segmentation_models_pytorch/losses/tversky.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         self.gamma = gamma
 
     def aggregate_loss(self, loss):
-        return loss.mean() ** self.gamma
+        return (loss ** self.gamma).mean()
```

Comment:
@zifuwanggg to fix black
```suggestion
        return (loss**self.gamma).mean()
```